python/tvm/target.py

```diff
@@ -114,6 +114,9 @@ def __init__(self,
         elif target_name in ("metal",):
             self.keys += ("gpu",)
             self.max_num_threads = 256
+        elif target_name in ("opengl"):
```

Comment:
Shouldn't it be `("opengl",)`? It ought to be searching for `target name` in a tuple.

Reply:
Fixed.